commonItems/ParadoxBool.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace commonItems {
+	public class ParadoxBool {
+		public bool Value { get; set; } = true;
+		public string YesOrNo {
+			get {
+				if (Value) {
+					return "yes";
+				}
+				return "no";
+			}
+		}
+		public ParadoxBool() {
+			Value = true;
```

Comment:
Is it necessary to set in the constructor if it's set in the definition above?

Reply:
No, it's redundant, removing it